src/shared/tools.h

```diff
@@ -1685,8 +1683,10 @@ extern int listzipfiles(const char *dir, const char *ext, vector<char *> &files)
 extern void backup(const char *fname, const char *ext, int revision = 0, int start = 1, bool store = false, bool full = true);
 
 extern void endianswap(void *, int, int);
-extern void seedMT(uint seed);
-extern uint randomMT();
+extern int rnd(int);
+extern float rndfloat(int);
+extern int detrnd(int, int);
+extern uint tmprnd();
```

Comment:
I assume the following mapping now, given their previous use:

- `rnd` -> `rnd`
- `rndscale` -> `rndfloat`
- `detrnd` -> `detrnd`
- `randomMT` -> `tmprnd`

In order to allow people who are used to the old code understand the new format, I think it makes sense to add docstrings to the new functions, explaining what the new ones do.